_scripts/webpack.main.config.js

```diff
@@ -15,12 +15,18 @@ const config = {
   module: {
     rules: [
       {
-        test: /\.js$/,
-        use: 'babel-loader',
+        test: /\.[tj]s$/,
+        loader: 'esbuild-loader',
         exclude: /node_modules/,
-      },
+        options: {
+          target: 'esnext'
```

Comment:
The electron version we are using still uses node 16, wouldn't esnext imply whatever the latest version of node is?

Probably best to specify the actual version of node we are using (the whole point of this is to not end up with code that doesn't work).

Reply:
I'll set this setting to `es2020`, [all of its functionalities](https://caniuse.com/sr_es11) are supported by majors browsers since many version.